### DISCUSSION TITLE
Allow owners to pin threads at the top above other threads!

### DISCUSSION THREAD

**Discussion by nikolqy** [community/community#3036] [Discussions]
Allow owners to pin threads at the top above other threads!

There should be a way for me to pin **Welcome to storm Discussions!** on top of **sdf** & **sdfdsf**

![image](https://user-images.githubusercontent.com/76059737/102668988-fea0b200-4152-11eb-80f8-66f463dce2fb.png)

**Comment by evi-liu** [comment]
Are you finding that users are having a tough time finding the pinned discussions as they are today?
